week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -89,12 +89,12 @@ function render() {
     });
 
     //add delete button to every row and render again
-    let delButton = document.createElement("button");
+    let delBut = document.createElement("button");
```

Comment:
👍

Reply:
I would actuall relabel all the other variables to be `delButton` here but it still works 👍